src/models/Bookmark.ts

```diff
@@ -0,0 +1,22 @@
+import { ObjectId } from "mongodb";
```

Comment:
rename file Bookmark.model.ts

Reply:
I did. but why does User.ts isn't named like this?